src/main/java/github/weichware10/toolbox/Main.java

```diff
@@ -1,86 +1,98 @@
 package github.weichware10.toolbox;
 
-import github.weichware10.toolbox.gui.ConfirmBox;
-import github.weichware10.toolbox.gui.Startbildschirm;
+import github.weichware10.toolbox.gui.App;
+import github.weichware10.toolbox.gui.ConfirmDialog;
+import io.github.cdimascio.dotenv.Dotenv;
+import java.io.File;
 import java.io.IOException;
 import java.io.OutputStream;
 import java.io.PrintStream;
 import javafx.application.Application;
 import javafx.event.EventHandler;
-import javafx.scene.Scene;
-import javafx.scene.control.TextArea;
 import javafx.scene.image.Image;
 import javafx.stage.Stage;
 import javafx.stage.WindowEvent;
+import org.joda.time.DateTime;
 
 /**
  * Toolbox GUI.
- *
  */
 public class Main extends Application {
+    /**
+     * Einstiegspunkt der Toolbox - Einführungsbildschirm.
+     *
+     * @param args - args
+     */
     public static void main(String[] args) {
+
+        // in Datei und Konsole loggen
+        String location = Dotenv.configure().directory(".").load().get("LOGS");
+        PrintStream ps = new PrintStream(new Console(location, System.out), true);
+        System.setOut(ps);
+        System.setErr(ps);
+
         launch(args);
     }
 
     @Override
-    public void start(Stage primaryStage) throws Exception {
-
-        // Konsole zum Debuggen
-        boolean debug = false;
-        if (debug) {
-            Stage logStage = new Stage();
-            TextArea textArea = new TextArea();
-            Console console = new Console(textArea);
-            PrintStream ps = new PrintStream(console, true);
-            System.setOut(ps);
-            System.setErr(ps);
-            Scene terminal = new Scene(textArea);
-            logStage.setScene(terminal);
-            logStage.show();
-        }
-
+    public void start(Stage primaryStage) {
 
-        //startet die erste Szene
-        Startbildschirm.display(primaryStage);
+        new App(primaryStage);
 
         // Event welches beim schließen eines Fensters aufgerufen wird
         primaryStage.setOnCloseRequest(new EventHandler<WindowEvent>() {
             @Override
             public void handle(WindowEvent event) {
-                // Wir kümmern uns selber um das schließen
-                event.consume();
                 // Fenster schließen, ja oder nein?
-                closeProgramm(primaryStage);
+                boolean confirmation = new ConfirmDialog("Do you want to close the window?").getConfirmation();
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 111).